dissect/target/plugins/os/windows/log/amcache.py

```diff
@@ -95,14 +115,18 @@ def check_compatible(self):
         if not self.logs.exists():
             raise UnsupportedPluginError("No amcache install logs found")
 
-    @export(record=DynamicDescriptor(["uri"]))
-    def amcache_install(self):
-        """Return the contents of the Amcache install log."""
+    @export(record=[AmcacheArpCreateRecord, AmcacheFileCreateRecord])
+    def amcache_install(self) -> Iterator[AmcacheArpCreateRecord, AmcacheFileCreateRecord]:
+        """Return the contents of the Amcache install log.
+
+        The log file contains the changes an installer performed on the system.
+        These only get created when the executable is an installer.
+        """
         for f in self.logs.iterdir():
             d = {}
             arp = []
             created = []
```

Comment:
```suggestion
            arp_created = []
            file_created = []
```

Maybe change this also while we're at it, makes it a bit more readable.